apis.go

```diff
@@ -58,6 +58,11 @@ type OVNDBApi interface {
 	// List Load balancers for a LSW
 	LSLBList(lswitch string) ([]*LoadBalancer, error)
 
+	// Add ExtIDs to SWITCH
+	LSExtIdsAdd(lswitch string, external_ids map[string]string) (*OvnCommand, error)
```

Comment:
Why separate method? Why not create LSSet and use it for setting all variables in feature that present in logical switch and not related to weak or strong references to other tables?

Reply:
@hzhou8 what you think?